### DISCUSSION TITLE
Indicate truncated messages (commit description) in log history (...)

### DISCUSSION THREAD

**Discussion by actondev** [magit/magit#5202] [Support] [answered]
Is it currently possible via some configuration to show in the log history if a commit message has more text in it? Similar to how github has this `...` . I think it's quite useful when browsing the commit history to find in a glance when there's something more to read.
![Screenshot_2024-08-15_02-27-52](https://github.com/user-attachments/assets/1671d625-9250-4ff6-8fce-273d491a1296)

**Comment by kyleam** [comment, marked as answer]
> Is it currently possible via some configuration to show in the log history if a commit message has more text in it?

No, there's not support for showing an indicator that there's a commit message body.

Here's some past discussion on this:

 * https://github.com/magit/magit/issues/4127
 * https://github.com/magit/magit/pull/4269
